content/md/connectivity/connection-dashboard.md

```diff
@@ -50,13 +50,23 @@ You can easily switch the destination connection you want to monitor by clicking
 # Error monitoring
 
 ## In the Connection dashboard
-In this dashboard, you can easily track **business errors** that occurred over the last 7 days. Each square represents a connection and allows you to access the list of related errors by clicking on it. It will redirect you to the connection `Monitoring` page.
+In this dashboard, you can easily track **product errors** that occurred over the last 7 days. 
+- The graph represents the number of product errors (business and technical), 
+- and connection cards display the number of business errors for the past week.
 
-![Number of business error per connection](../img/business-errors-per-connection.png)
+::: info
+Julia could solve **business errors** inside the PIM or in the connected application. On the contrary, **technical errors** can't be resolved without a developer or a technical profile intervention.
```

Comment:
```suggestion
Julia could solve **business errors** from the PIM or in the connected application. On the contrary, **technical errors** can't be resolved without a developer or a technical profile intervention.
```

Reply:
Could -> tu veux parler de la possibilité ou de la probabilité ? 'Julia can solve' is probably a better option here.

Reply:
Elle peut résoudre mais elle peut choisir de ne pas le faire dans le PIM parce que l'erreur vient de l'application source, ou bien parce que le connecteur doit être corrigé...